src/desktop.md

```diff
@@ -315,6 +315,36 @@ from a Flutter desktop app, see the [Desktop Photo Search sample][].
 [msix package]: {{site.pub}}/packages/msix
 [Desktop Photo Search sample]: {{site.github}}/flutter/samples/tree/master/experimental/desktop_photo_search
 
+###### Create a .pfx certificate.
+
+For private deployment and testing with the help of MSIX Installer,
+you need to give your app a digital signature in the form of a .pfx certificate.
+
+For public deployment(Windows Store), .pfx certificate is not required.
```

Comment:
I'd suggest something like the following:

For deployment through the Windows Store, generating a .pfx certificate is not required. The Windows Store handles creation and management of certificates for applications distributed through the Windows Store.